Core/SVS/src/drawer_asio.cpp

```diff
@@ -0,0 +1,186 @@
+#include <iostream>
+#include <cstdlib>
+#include <asio.hpp>
+#include "drawer.h"
+#include "common.h"
+#include "sgnode.h"
+
+/**** 
+ * Note: This is a new implementation of the drawer socket that uses the asio library
```

Comment:
Could you quickly describe what the drawer socket is, or just link to another file that uses the term? It's un-googleable because of overlap with physical tools.

Reply:
I guess this is an SGEL thing? I dunno. The file should be described in a more standalone way, I think, rather than referring to an old thing that is going to be deleted after we accept the CMake build.